### DISCUSSION TITLE
Plex QSV (Intel GPU) hardware transcode not working

### DISCUSSION THREAD

**Discussion by smtdev** [tteck/Proxmox#1090] [Help Needed]
Months ago (oct 2022) I smoothly installed Plex LXC. There were no issues during this period using direct connection until today, when I needed to use the GPU transcoding, and it didn't work. I didn't test it before. 

I can see the lines below inside the LXC `.conf` file:

```
lxc.cgroup2.devices.allow: c 226:0 rwm
lxc.cgroup2.devices.allow: c 226:128 rwm
lxc.cgroup2.devices.allow: c 29:0 rwm
lxc.mount.entry: /dev/fb0 dev/fb0 none bind,optional,create=file
lxc.mount.entry: /dev/dri dev/dri none bind,optional,create=dir
lxc.mount.entry: /dev/dri/renderD128 dev/renderD128 none bind,optional,create=file
lxc.mount.entry: /hulk1/trunk/media mnt/media none rbind,create=dir,optional 0 0
```

What I noticed is that the GPU drivers were not installed in the host server. I solved it by running `apt-get install i965-va-driver` and restarted the Plex LXC, but no luck. 

Plex transcode options in Settings are enabled already.

Am I missing something? Should I execute or modify something else?

Thanks!

**Comment by ghost** [comment]
I have a similar issue, installed via the LXC script without any issues - transcoding seems to be working but nowhere in the dashboard does it indicate that it's hardware accelerated. Also do not see any increase in CPU utilisation. I have i5 10400 and GTX 1650 Super (not sure if it's maybe using this as render device)

`lxc.cgroup2.devices.allow: a
lxc.cap.drop:
lxc.cgroup2.devices.allow: c 226:0 rwm
lxc.cgroup2.devices.allow: c 226:128 rwm
lxc.cgroup2.devices.allow: c 29:0 rwm
lxc.mount.entry: /dev/fb0 dev/fb0 none bind,optional,create=file
lxc.mount.entry: /dev/dri dev/dri none bind,optional,create=dir
lxc.mount.entry: /dev/dri/renderD128 dev/dri/renderD128 none bind,optional,create=file`

is in my config file
